flink/src/main/scala/ru/itclover/tsp/io/input/InfluxDBInputConf.scala

```diff
@@ -97,6 +98,20 @@ case class InfluxDBInputConf(
     new TimeExtractor[Row] {
       override def apply(event: Row) = {
         val isoTime = getRowFieldOrThrow(event, fieldsIdxMap, dtField).asInstanceOf[String]
+        if (isoTime == null) sys.error(s"Time was null (tried field $dtField with " +
```

Comment:
It can be when `fieldsIdxMap` is wrong or corrupted, so this check was added to improve the error message (otherwise it simply says `NullPointerException: text` without even mentioning which text it is)